src/Tasks/Common/Resources/Strings.resx

```diff
@@ -702,4 +702,60 @@ The following are names of parameters or literal values and should not be transl
     <comment>{StrBegin="NETSDK1149: "}</comment>
   </data>
+  <data name="CrossgenToolMissingInPDBCompilationMode" xml:space="preserve">
+    <value>NETSDK1150: CrossgenTool not specified in PDB compilation mode.</value>
+    <comment>{StrBegin="NETSDK1150: "}</comment>
+  </data>
+  <data name="Crossgen2ToolMissingWhenUseCrossgen2IsSet" xml:space="preserve">
+    <value>NETSDK1151: Crossgen2Tool must be specified when UseCrossgen2 is set to true.</value>
+    <comment>{StrBegin="NETSDK1151: "}</comment>
+  </data>
+  <data name="Crossgen2ToolExecutableNotFound" xml:space="preserve">
+    <value>NETSDK1152: Crossgen2Tool executable '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1152: "}</comment>
+  </data>
+  <data name="DotNetHostExecutableNotFound" xml:space="preserve">
+    <value>NETSDK1153: .NET host executable '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1153: "}</comment>
+  </data>
+  <data name="JitLibraryNotFound" xml:space="preserve">
+    <value>NETSDK1154: JIT library '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1154: "}</comment>
+  </data>
+  <data name="Crossgen2MissingRequiredMetadata" xml:space="preserve">
+    <value>NETSDK1155: Required '{0}' metadata missing on Crossgen2Tool item.</value>
+    <comment>{StrBegin="NETSDK1155: "}</comment>
+  </data>
+  <data name="CrossgenToolMissingWhenUseCrossgen2IsNotSet" xml:space="preserve">
+    <value>NETSDK1156: CrossgenTool must be specified when UseCrossgen2 is set to false.</value>
+    <comment>{StrBegin="NETSDK1156: "}</comment>
+  </data>
+  <data name="CrossgenToolExecutableNotFound" xml:space="preserve">
+    <value>NETSDK1157: CrossgenTool executable '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1157: "}</comment>
+  </data>
+  <data name="DiaSymReaderLibraryNotFound" xml:space="preserve">
+    <value>NETSDK1158: DiaSymReader library '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1158: "}</comment>
+  </data>
+  <data name="PDBGeneratorInputExecutableNotFound" xml:space="preserve">
+    <value>NETSDK1159: PDB generation: R2R executable '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1159: "}</comment>
+  </data>
+  <data name="InputAssemblyNotFound" xml:space="preserve">
+    <value>NETSDK1160: Input assembly '{0}' not found.</value>
+    <comment>{StrBegin="NETSDK1160: "}</comment>
+  </data>
+  <data name="MissingOutputPDBImagePath" xml:space="preserve">
+    <value>NETSDK1161: Missing output PDB path in PDB generation mode (OutputPDBImage metadata).</value>
+    <comment>{StrBegin="NETSDK1161: "}</comment>
+  </data>
+  <data name="MissingOutputR2RImageFileName" xml:space="preserve">
+    <value>NETSDK1162: Missing output R2R image path (OutputR2RImage metadata).</value>
+    <comment>{StrBegin="NETSDK1162: "}</comment>
+  </data>
+  <data name="Crossgen5CannotEmitSymbolsInCompositeMode" xml:space="preserve">
+    <value>NETSDK1163: Crossgen2 5.0 doesn't support emitting symbols in composite mode</value>
```

Comment:
It misses a period at the end. Maybe rephrase it a little?
> Cannot emit symbols when publishing for .NET 5 with Crossgen2 using composite mode.

I assume if we ever have .NET 5.1, it will not support this scenario either.

Reply:
Good point, fixed in 16th commit. I have also fixed one other bug - CG2, just like CG1, expects just the directory names, not complete file names, in <code>--pdb-path</code> and <code>--perfmap-path</code> arguments, and that was handled incorrectly in PrepareForReadyToRunCompilation. This will ultimately need a CG2 fix to put perfmap naming in sync with what PrepareForR2RC expects, that is not the case today.